### PR TITLE
Hw3 oleksandr petrov

### PR DESCRIPTION
**Pull request by PetrovAI-165**

### CODE REVIEW

File: Oleksadr_Petrov/HW3/Task5.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+                   3: common_guests.pop(),
+                   4: common_guests.pop(),
+                   5: common_guests.pop(),
+                   6: None,
+                   7: None,
+                   8: None
+             }
+
+print(vip_places)
+print(common_places)
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW3/Task5.py:21]: vip_places is a mutable type. I can add another guest!

File: Oleksadr_Petrov/HW3/Task6.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+names = ["Minodora Áurea", "Itoro Alban", "Gabriella Emil", "Sigmund Govinda", "Minodora Áurea", "Itoro Alban"]
+result = []
+for name in names:
+    if name not in result:
+        result.append(name)
```
Comment (bohdandovbsyh) [inline Oleksadr_Petrov/HW3/Task6.py]: let it be. 
But try to solve it using fromkeys() function

Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW3/Task6.py]: changes applied

File: Oleksadr_Petrov/HW3/Task5.py (hunk 1 above)
Comment (PetrovAI-165) [reply Oleksadr_Petrov/HW3/Task5.py:21]: changes applied

**Review by bohdandovbsyh — APPROVED** [review]

**Review by bohdandovbsyh — APPROVED** [review]

**Comment by bohdandovbsyh** [comment]
@PetrovAI-165 please fix the conflicts

**Pull request merged**